src/lwc/geFormService/geFormService.js

```diff
@@ -0,0 +1,70 @@
+import getRenderWrapper from '@salesforce/apex/GE_TemplateBuilderCtrl.retrieveDefaultFormRenderWrapper';
+
+const inputTypeByDescribeType = {
```

Comment:
More of a QA item (and I know this is just a first pass!), but we're missing PERCENT here, which I guess could just be number for now?

Reply:
PERCENT should now render. I had it added as a valid lightning-input type, but it didn't have a mapping in this object. It does now, and will render as a number lightning-input.